ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/adopt_etc/tasks/main.yml

```diff
@@ -115,19 +115,3 @@
     state: present
     backup: yes
   tags: hosts_file, adoptopenjdk
-
-# This doesn't work as ansible says it can't write to the directory
-# Leaving this here in case someoen can make it work in the future
-#
-#- name: Record playbook completion with time in /var/log/ansiblerun.log
-#  lineinfile:
-#    state: present
-#    create: yes
-#    path: /var/tmp/ansiblerun.log
-#    mode: 0644
-#    line: "{{ ansible_date_time.iso8601_basic_short }} END"
-#  tags: ansiblerunlog
-
-# Version of the above that works
-- name: Record playbook completion with time in /var/log/ansiblerun.log
```

Comment:
I think we should consider having this as a "starting to run playbook" whereas your `logs` role is marking it as complete i.e. they could both write to the same file, which would allow us to see from the single log file if an ansible run was attempted but didn't succeed (Since `logs` is at the end that's a good indication that it got to the end of the playbook)

Reply:
Done. And i've changed the time variable to use the `date` command instead since ansible's time variable is calculated at he beginning and is not updated during playbook execution